engine/behaviors.js

```diff
@@ -123,6 +123,8 @@
 
 			function fireBullet(xDirection, yDirection, tFrame){
 		      //console.log('firing bullet', xDirection, yDirection);
+					LP.audioEngine.trigger("fire");
```

Comment:
Wonder how is the effect when the circleEnemy fires 18 bullets at the same time! Let's check it out...